### PR TITLE
Misleading error on invalid meshconfig

### LINKED ISSUE #5752: [mesheryctl] Misleading error on invalid meshconfig
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior

If I have an ill-formatted config.yaml inside .meshery directory and I try to execute any mesheryctl command, it results in the error shown below which is misleading as the actual issue the invalid yaml
Screenshot 2022-07-04 at 9 58 49 PM
Screenshot 2022-07-04 at 9 59 32 PM

#### Environment

- Host OS: Mac Linux Windows
- Meshery Server Version: stable-v
- Meshery Client Version: stable-v
- Platform: Docker Kubernetes

---

#### Contributor Resources
- [mesheryctl Contributing Guide](https://github.com/meshery/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by manikmmalhotra**
Signed-off-by: manikmmalhotra <manikmalhotra2113@gmail.com>

**Description**
If I have an ill-formatted config.yaml inside .meshery directory and I try to execute any mesheryctl command, it results in the misleading error, fixed it by 'Invalid meshconfig'
This PR fixes #5752 
Screenshot from 2022-07-20 16-05-01

**Notes for Reviewers**
Always open for suggestions and improvements. Please point me out if anything should be added/corrected.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by manikmmalhotra** [comment]
@Revolyssup need your reviews here.

**Comment by leecalcote** [comment]
Thanks, @manikmmalhotra. I don't know that this reformatted error message offers any more clarity.

File: mesheryctl/internal/cli/root/config/config.go [hunk 1]
```diff
@@ -69,7 +69,7 @@ func UpdateContextInConfig(v *viper.Viper, context *Context, name string) error
 // CheckIfCurrentContextIsValid checks if current context is valid
 func (mc *MesheryCtlConfig) CheckIfCurrentContextIsValid() (*Context, error) {
 	if mc.CurrentContext == "" {
-		return &Context{}, errors.New("current context not set")
+		return &Context{}, errors.New("Invalid meshconfig")
```
Comment (hexxdump) [inline mesheryctl/internal/cli/root/config/config.go:72]: How about the below error?
Valid context is not available in meshconfig

**Review by leecalcote — APPROVED** [review]
Thank you, @hexxdump @manikmmalhotra

**Pull request merged**